robottelo/constants.py

```diff
@@ -241,7 +241,7 @@
 DEFAULT_CV = "Default Organization View"
 DEFAULT_PTABLE = "Kickstart default"
 DEFAULT_SUBSCRIPTION_NAME = (
-    'Red Hat Enterprise Linux Server Entry Level, Self-support')
+    'Red Hat Enterprise Linux Server, Premium (Physical or Virtual Nodes)')
```

Comment:
Yeah. Well as i already have to rerun hostcollection tests i think that's the way we'll validate the change. Anyway it works perfectly with existing contenthost registering/errata installation tests, and right now all the subscriptions-related tests are failing on jenkins, so this subscription should be a good candidate.